### PR TITLE
LPS-165345 Expose Cookie types

### PR DESCRIPTION
**Pull request by victorg1991**
This PR exposes the cookie types in the Liferay.Util global variable to be able to use from anywhere

### CODE REVIEW

**Comment by eltonccdantas** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/tabs/index.js [hunk 1]
```diff
@@ -32,11 +32,18 @@ const persistedTab = (function () {
 
 	return {
 		getId() {
-			return Number(sessionStorage.getItem(persistedTabKey));
+			return Number(
+				Liferay.Util.SessionStorage.getItem(persistedTabKey),
+				Liferay.Util.COOKIE_TYPES.PERSONALIZATION
+			);
```
Comment (markocikos) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/tabs/index.js:38]: Does this do anything? `Number` has a single argument, the second one is ignored.

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js [hunk 2]
```diff
@@ -334,6 +335,8 @@ Liferay.Util.unescapeHTML = unescapeHTML;
 
 Liferay.Util.Cookie = Cookie;
 
+Liferay.Util.COOKIE_TYPES = COOKIE_TYPES;
```
Comment (markocikos) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js:338]: We are already exposing consent types, as `Liferay.Util.SessionStorage.TYPES`. See example usage [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/js/OAuth2Client.ts#L104-L107).

**Review by markocikos — CHANGES_REQUESTED** [review]
@victorg1991 Please see comments inline.

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js (hunk 2 above)
Comment (victorg1991) [reply modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js:338]: Upps didn't notice that 😱

**Pull request closed without merging**